poisson_lib.py

```
@@ -208,11 +227,34 @@ def analyze_poisson_outcome(
         # ✅ Consolidated fmt_p import: Use logic.py's centralized formatting
-        from logic import (clean_numeric_value, _robust_sort_key, get_label, 
-                           fmt_p, fmt_p_with_styling)
+        from logic import (
+            _robust_sort_key,
+            clean_numeric_value,
+            fmt_p,
+            fmt_p_with_styling,
+            get_label,
+        )
         
         logger.info(f"Starting Poisson analysis for outcome: {outcome_name}")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)